src/dialog/dlgreplacecuecolor.h

```diff
@@ -66,6 +67,6 @@ class DlgReplaceCueColor : public QDialog, public Ui::DlgReplaceCueColor {
     parented_ptr<WColorPickerAction> m_pCurrentColorPickerAction;
     mixxx::RgbColor::optional_t m_lastAutoSetNewColor;
     mixxx::RgbColor::optional_t m_lastAutoSetCurrentColor;
-    QStyle* m_pStyle;
+    std::unique_ptr<QStyle> m_pStyle;
```

Comment:
This member should be `const` to prevent that the memory is freed after handing it to the widgets.

Reply:
How does that prevent freeing the memory? IMO instead the pointer should be explicitly `.release()`d at definite last use.